docs/spark/spark-writes.md

```diff
@@ -328,14 +330,30 @@ PARTITIONED BY (days(ts), category)
 
 To write data to the sample table, your data needs to be sorted by `days(ts), category`.
 
-If you're inserting data with SQL statement, you can use `ORDER BY` to achieve it, like below:
+If you're inserting data with SQL statement, you can use either `ORDER BY` to trigger global sort, or `SORT BY` to trigger local sort. Global sort like below:
 
 ```sql
 INSERT INTO prod.db.sample
 SELECT id, data, category, ts FROM another_table
 ORDER BY ts, category
 ```
 
+Local sort like below:
+
+```sql
+INSERT INTO prod.db.sample
+SELECT id, data, category, ts FROM another_table
+SORT BY ts, category
+```
+
+Of course, you can also use `SORT BY` with partition transforms.
+
+```sql
+INSERT INTO prod.db.sample
+SELECT id, data, category, ts FROM another_table
+SORT BY days(ts), category
```

Comment:
Should we register the `days(..)` UDF into spark environment firstly before we call it ?  I will suggest to add the SQL about how to register the `days(..)` UDF.

Reply:
I perfected the description, what do you think of this description?